app.py

```diff
@@ -47,11 +48,25 @@ def sync(force_sync: bool, graceful: bool, debug: bool) -> None:
     graph = GraphDatabase()
     graph.connect()
 
+    # First we need to sync solvers
+
+    stats = sync_solver_documents(force=force_sync, graceful=graceful, graph=graph)
+
+    _LOGGER.info(
+        "Syncing triggered by %r function completed with "
+        "%d processed, %d synced, %d skipped and %d failed documents",
+        *stats,
+    )
+
     for obj_name, function in thoth_sync_module.__dict__.items():
         if not obj_name.startswith("sync_"):
             _LOGGER.debug("Skipping attribute %r of thoth-storages syncing module: not a syncing function", obj_name)
             continue
 
+        if not obj_name.startswith("sync_solver"):
```

Comment:
```python
        if obj_name == "sync_solver_documents"
```

Reply:
BTW the name `sync_solver_result`?

https://github.com/thoth-station/storages/blob/998abf998da7c8e486e502822e2a9bfdf67f53d2/thoth/storages/graph/postgres.py#L4813

Reply:
Ah, never mind, it's from sync module :)